### PR TITLE
Refactor/add sad paths to tests

### PR DESCRIPTION
**Pull request by Eleanorgruth**
Summary of the work done:

Add sad paths to RecipeRepository for the filterTag and filterName methods
This branch includes:

I also changed the double quotes to single quotes
Added a test per Kiko's advice to check what happens  if an empty string is passed into the filter
Contributed to code:

Eleanor
Needs to review:

Kiko and Jenn
Additional notes about any issues/questions remaining around this code:

Can you double-check to make sure that all sad paths are tested?

### CODE REVIEW

**Review by knpausch — APPROVED** [review]
Awesome! Yay sad paths!

File: test/RecipeRepository-test.js [hunk 1]
```diff
@@ -28,11 +28,22 @@ describe('RecipeRepo', () => {
...
-    expect(recipeRepository.filterName("loaded chocolate chip pudding cookie cups")).to.deep.equal([recipe1])
+    expect(recipeRepository.filterName('maple dijon apple cider grilled pork chops')).to.deep.equal([recipe2])
+    expect(recipeRepository.filterName('loaded chocolate chip pudding cookie cups')).to.deep.equal([recipe1])
+  })
+  it('Should return an empty array if the name searched does not match any recipes.', () => {
+    expect(recipeRepository.filterName('calamari')).to.deep.equal([])
+    expect(recipeRepository.filterName('hamburger')).to.deep.equal([])
+  })
+  it('Should return all recipes if an empty string is passed in.', () => {
+    expect(recipeRepository.filterName('')).to.deep.equal([recipe1, recipe2])
```
Comment (jmyacobn) [inline test/RecipeRepository-test.js:47]: Hi Eleanor, the code I just implemented for the search bar error message says that if the user puts in an empty string, the error message will pop up. Should we change this to give an empty array or should I go back and change the code?

**Review by jmyacobn — CHANGES_REQUESTED** [review]
Approved after changes made to the last sad path!

**Review by jmyacobn — APPROVED** [review]
As discussed in the standup, Eleanor's sad path test does accurately reflect the data model. My message to users does not affect that, so no changes needed to merge!

**Pull request merged**